static/skywire-manager-src/src/assets/i18n/en.json

```diff
@@ -375,6 +375,7 @@
       "vpn-title": "VPN-Server Settings",
       "new-password": "New password (Leave empty to remove the password)",
       "repeat-password": "Repeat password",
+      "netifc": "netifc (optional)",
```

Comment:
Hey buddy. Could you change its title to `Default Network Interface`? [also for es and es_base]